layouts/index.html

```diff
@@ -27,5 +27,5 @@ <h3 class = 'article_title'>{{ $value.Title }}</h3>
   {{- if and (eq $index (add $size -1)) (gt $size 1) }}</div>{{ end }}
   {{- end }}
 </div>
-<a href = '{{ absURL (printf "post/%s" "") }}' class = 'post_nav'><span class = 'post_next'>View Archive</span></a>
+<a href = '{{ absURL (printf "posts/%s" "") }}' class = 'post_nav'><span class = 'post_next'>View Archive</span></a>
```

Comment:
@plamen, thanks for contributing. This commit will break things while working with the [exampleSite](https://github.com/onweru/newsroom/tree/master/exampleSite/content). As you can see, the posts directory is named `post` in that case. To solve this issue for good, perhaps there is a way not to hard code the directory into the path. I will check and proceed accordingly.

Reply:
done, removed the commit